apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,22 @@
+import NextAuth from 'next-auth'
+import GithubProvider from 'next-auth/providers/github'
+
+export default NextAuth({
+  // Configure one or more authentication providers
+  providers: [
+    GithubProvider({
```

Comment:
https://github.com/organizations/testing-accessibility/settings/apps/testingaccessibility-com-auth is where this got set up at and you'd need to fix up the env for it to function locally